### PR TITLE
execute_dotnet_assembly Some fix for rubocop verification

### PR DESCRIPTION
**Pull request by b4rtik**
This PR to fix some problems with rubocop verification

### CODE REVIEW

File: modules/post/windows/manage/execute_dotnet_assembly.rb [hunk 1]
```diff
@@ -289,7 +279,8 @@ def read_output(process)
         break if output.nil? || output.empty?
       end
     rescue Rex::TimeoutError => e
-    rescue ::Exception => e
+      vprint_warning('Time out exception: wait limit exceeded (5 sec)')
+    rescue ::StandardError => e
```
Comment (adfoster-r7) [inline modules/post/windows/manage/execute_dotnet_assembly.rb:283]: Looks like a good catch from Rubocop! :+1:

**Comment by b4rtik** [comment]
I noticed that the bypass for etw no longer works properly and the process crashes. I have yet to elaborate but I think the reason is a windows update.
I think I can find a solution. But I still have the impression that this is too volatile at the moment and that the default value for the etwbypass flag should be false.

**Comment by b4rtik** [comment]
I verified that there were no changes in the implementation of the EtwEventWrite function so the error was not caused by the recent update.
I found a bug in the patching function.

**Comment by bwatters-r7** [comment]
```
msf5 post(windows/manage/execute_dotnet_assembly) > show options

Module options (post/windows/manage/execute_dotnet_assembly):

   Name            Current Setting  Required  Description
   ----            ---------------  --------  -----------
   AMSIBYPASS      true             yes       Enable Amsi bypass
   ARGUMENTS                        no        Command line arguments
   DOTNET_EXE                       yes       Assembly file name
   ETWBYPASS       true             yes       Enable Etw bypass
   PID             0                no        Pid  to inject
   PPID            0                no        Process Identifier for PPID spoofing when creating a new process. (0 = no PPID spoofing)
   PROCESS         notepad.exe      no        Process to spawn
   SESSION                          yes       The session to run this module on.
   USETHREADTOKEN  true             no        Spawn process with thread impersonation
   WAIT            10               no        Time in seconds to wait

msf5 post(windows/manage/execute_dotnet_assembly) > set session 1
session => 1
msf5 post(windows/manage/execute_dotnet_assembly) > set verbose true
verbose => true
msf5 post(windows/manage/execute_dotnet_assembly) > set dotnet_exe '/home/tmoose/rapid7/Seatbelt/Seatbelt/bin/Debug/Seatbelt.exe' 
dotnet_exe => /home/tmoose/rapid7/Seatbelt/Seatbelt/bin/Debug/Seatbelt.exe
msf5 post(windows/manage/execute_dotnet_assembly) > set arguments user
arguments => user
msf5 post(windows/manage/execute_dotnet_assembly) > run

[*] Dot Net Versions installed on target: ["2.0.50727.4927", "3.0.30729.4926", "3.5.30729.4926", "4.7.03056", "4.0.0.0"]
[*] CLR versione required v2.0.50727
[*] Requirements ok
[*] Running module against DESKTOP-D1E425Q
[*] Launching notepad.exe to host CLR...
[+] Process 7864 launched.
[*] Reflectively injecting the Host DLL into 7864..
[*] Injecting Host into 7864...
[*] Host injected. Copy assembly into 7864...
[*] Assembly copied.
[*] Executing...
[*] Start reading output
[+] 
[+] 
[+]                         %&&@@@&&                                                                                  
[+]                         &&&&&&&%%%,                       #&&@@@@@@%%%%%%###############%                         
[+]                         &%&   %&%%                        &////(((&%%%%%#%################//((((###%%%%%%%%%%%%%%%
[+] %%%%%%%%%%%######%%%#%%####%  &%%**#                      @////(((&%%%%%%######################(((((((((((((((((((
[+] #%#%%%%%%%#######%#%%#######  %&%,,,,,,,,,,,,,,,,         @////(((&%%%%%#%#####################(((((((((((((((((((
[+] #%#%%%%%%#####%%#%#%%#######  %%%,,,,,,  ,,.   ,,         @////(((&%%%%%%%######################(#(((#(#((((((((((
[+] #####%%%####################  &%%......  ...   ..         @////(((&%%%%%%%###############%######((#(#(####((((((((
[+] #######%##########%#########  %%%......  ...   ..         @////(((&%%%%%#########################(#(#######((#####
[+] ###%##%%####################  &%%...............          @////(((&%%%%%%%%##############%#######(#########((#####
[+] #####%######################  %%%..                       @////(((&%%%%%%%################                        
[+]                         &%&   %%%%%      Seatbelt         %////(((&%%%%%%%%#############*                         
[+]                         &%%&&&%%%%%        v0.2.0         ,(((&%%%%%%%%%%%%%%%%%,                                 
[+]                          #%%%%##,                                                                                 
[+] 
[+] 
[+] 
[+] === Running User Triage Checks ===
```

**Pull request merged**